docs/protocol/substrate-cli.mdx

```diff
@@ -902,19 +902,23 @@ There are extra commands and parameters you can use on farmer or node, use the `
 
 ---
 
-### Helpful commands examples
+### Node & Farmer Command Guide
```

Comment:
Command -> Commands

Reply:
I also think that making the header larger by using ## instead of ### is better here, since all next subsections refer to the commands guide.